src/styles/index.scss

```diff
@@ -1,3 +1,74 @@
-body {
+@import './variables';
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  height: 100vh;
+}
```

Comment:
According to the BEM methodology, `.body` should be named with a more specific class name that reflects its role as a block. For example, `.page-container` or `.main-container` could be more descriptive and align with BEM naming conventions.